docs/v0.0.1/layout-specs.md

```diff
@@ -1,18 +1,57 @@
 # Layout Specifications
-The layout specification is a `json` document that describes the layout of the library view. It outlines the hierarchical structure (i.e. tree structure) of the library view, as well as defines items that should go under each tree node.
+The layout specification is a `json` document that describes the layout of the library view. It outlines the hierarchical structure (i.e. tree structure) of the library view, as well as defines items that should go under each tree node. The layout specification works closely with `loadedTypes` JSON data, obtained from [Loaded Data Types](./loaded-data-types.md).
 
-### The simplest form of element
-The following example illustrates the simplest form of a specification `json` document -- it is made up of a single *element* (the layout specification document is a hierarchical tree of *elements*). The `elements` entry represents a list of root elements that should be displayed at the root level of the library view. For this example, the library view contains only a single root node with `Display` as its text:
```

Comment:
If I may rephrase this part, I would suggest:

*At the root of the layout specification `json` document is a single `sections` element. This `sections` element contains a list of sections that should be displayed at the root level of the library view*